install/windows/viewer.spec

```diff
@@ -148,7 +181,7 @@ a = Analysis(
     datas=None,
     hiddenimports=[
         'cairo',
-        'Queue',
+        #'Queue',  # not required for python3 installation
         'queue',
```

Comment:
Please make it still compatible with Python2.
I'd suggest that you move all the "queue" related imports to a separate function. You can detect the python version from the current interpreter.